docusaurus/docs/reference/ha/certificates.md

```diff
@@ -0,0 +1,86 @@
+---
+sidebar_label: Certificates
+sidebar_position: 20
+---
+
+# Controller Certificates
+
+For controllers to communicate and trust one another, they need certificates that have
+been generated with the correct attribute and relationships.
+
+## Requirements
+
+1. The certificates must have a shared root of trust
+2. The controller client and server certificates must contain a 
+   [SPIFFE ID](https://spiffe.io/docs/latest/spiffe-about/spiffe-concepts/#spiffe-id)
+
+## Steps to Certificate Creation
+There are many ways to set up certificates, so this will just cover a recommended configuration.
+
+The primary thing to ensure is that controllers have a shared root of trust. 
+A standard way of generating certs would be as follows:
+
+1. Create a self-signed root CA
+1. Create an intermediate signing cert for each controller
+1. Create a server cert using the signing cert for each controller
+1. Create a client cert using the signing cert for each controller
+1. Make sure that the CA bundle for each server includes both the root CA and the intermediate CA
```

Comment:
> Make sure that the CA bundle for each server includes both the root CA and the intermediate CA for that server

This needs specifics. Open the controller config file, fine the 'ca' line, add the intermediary CA... Also this instruciton conflcts with our latest best practice of using 'only root cas' in the ca bundle. This might need discussion?

I would think the client should just send the leaf/intermediate chain when connecting though?

Reply:
That's what I expected to see, that leaves must be presented with their issuer chains, not that intermediates must appear in the root CA bundle, since they're not roots, and treating them like roots defeats the purpose of an intermediate.